### PR TITLE
feature/bash

### PR DESCRIPTION
**Pull request by 260677**

### CODE REVIEW

**Comment by manuelbarzi** [comment]
REDO this branch. you have added extra folders (helloword...) that should not be here.

**Comment by iConceali** [comment]
You also have the Challenge folder added

File: staff/carolina-machado/bash/README-1.md
Comment (fercostacastro) [inline staff/carolina-machado/bash/README-1.md:1]: all the explanation should be in English

Comment (260677) [reply staff/carolina-machado/bash/README-1.md:1]: hi Fe, thanks for the comment!
all the descriptions are now in english, please check back and give me a feedback (and maybe a "like" too 😇 )
thank you!!

**Comment by 260677** [comment]
> You also have the Challenge folder added

hi Adri, thanks for the comment!
it's all fixed now, also made it nicer, please check back and give me a feedback (and maybe a "like" too 😇 )
thank you!!

**Comment by 260677** [comment]
> REDO this branch. you have added extra folders (helloword...) that should not be here.

hi Manu, thanks for the comment!
it's all fixed now, also made it nicer, please check back and give me a feedback (and maybe a "like" too 😇 )
thank you!!

File: staff/carolina-machado/bash/README.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
+# BASH
+
+Programación en Bash – Programación para la bioinformática
```
Comment (manuelbarzi) [inline staff/carolina-machado/bash/README.md]: translate to english (fer asks)

Comment (260677) [reply staff/carolina-machado/bash/README.md]: Done! :)

File: staff/carolina-machado/bash/README.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+```
+
+## tree <path>
+shows files\folders structure in tree mode
+```sh
+$ tree workspace/isdi-bootcamp-202402
+```
+
+## rm -rf 
+remove folder and all its content
```
Comment (manuelbarzi) [inline staff/carolina-machado/bash/README.md:90]: add a demo code block

File: staff/carolina-machado/bash/README.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+access a different branch
+```sh
+$ git checkout playground
+
+git checkout feature/playground
+Switched to branch 'feature/playground'
+Your branch is up to date with 'origin/feature/playground'.
+```
+
+## git push
```
Comment (manuelbarzi) [inline staff/carolina-machado/bash/README.md]: move git commands to git manual (another issue and branch)

Comment (260677) [reply staff/carolina-machado/bash/README.md]: done!

File: staff/carolina-machado/bash/README.md (hunk 2 above)
Comment (260677) [reply staff/carolina-machado/bash/README.md:90]: done :)

**Pull request merged**